kfs-ld/src/main/resources/edu/arizona/kfs/module/ld/spring-ld.xml

```diff
@@ -24,6 +19,39 @@
         </property>
     </bean>
 
+    <bean id="laborGLScrubberValidator" parent="laborGLScrubberValidator-parentBean" />
+    <bean id="laborGLScrubberValidator-parentBean" class="edu.arizona.kfs.module.ld.batch.service.impl.LaborGLScrubberValidatorImpl" parent="scrubberValidator" abstract="true" />
+
+    <bean id="laborScrubberValidator" parent="laborScrubberValidator-parentBean" />
+    <bean id="laborScrubberValidator-parentBean" class="edu.arizona.kfs.module.ld.batch.service.impl.ScrubberValidatorImpl" abstract="true">
+        <property name="scrubberValidator" ref="laborGLScrubberValidator"/>
+        <property name="configurationService" ref="kualiConfigurationService"/>
+        <property name="accountService" ref="accountService"/>
+        <property name="persistenceService" ref="persistenceService"/>
+        <property name="persistenceStructureService" ref="persistenceStructureService"/>
+        <property name="parameterService" ref="parameterService"/>
+        <property name="businessObjectService" ref="businessObjectService"/>
+        <property name="balanceTypService" ref="balanceTypService"/>
+        <property name="optionsService" ref="optionsService"/>
+        <property name="globalTransactionEditService" ref="globalTransactionEditService"/>
+    </bean>
+
+    <bean id="salaryTransferPeriodValidationService" parent="salaryTransferPeriodValidationService-parentBean" />
+    <bean id="salaryTransferPeriodValidationService-parentBean" class="edu.arizona.kfs.module.ld.batch.service.impl.SalaryTransferPeriodValidationServiceImpl" abstract="true">
+        <property name="effortCertificationService" ref="effortCertificationModuleService"/>
+        <property name="documentService" ref="documentService"/>
+        <property name="noteService" ref="noteService"/>
+        <property name="configurationService" ref="kualiConfigurationService"/>
+        <property name="globalTransactionEditService" ref="globalTransactionEditService"/>
+    </bean>
+
+    <bean id="globalTransactionEditService" parent="globalTransactionEditService-parentBean" />

```

Comment:
Sure, the service being a singleton will point to the same thing, I just figured it might be a convenience to have it be explicitly identified. Removed anyway, which will be another reason for devs to correctly configure their IDE for spring contexts to follow references, which we should all be doing anyways.